packages/sdk/src/channelsWithEntitlements.test.ts

```diff
@@ -36,178 +37,16 @@ import {
     Operation,
     CheckOperationType,
     treeToRuleData,
-    ISpaceDapp,
     encodeThresholdParams,
     createExternalNFTStruct,
 } from '@river-build/web3'
-import { Client } from './client'
 import { make_MemberPayload_KeySolicitation } from './types'
 
 const log = dlog('csb:test:channelsWithEntitlements')
 const twoEth = BigInt(2e18)
 const oneEth = BigInt(1e18)
 const oneHalfEth = BigInt(5e17)
 
-// pass in users as 'alice', 'bob', 'carol' - b/c their wallets are created here
-async function setupChannelWithCustomRole(
```

Comment:
This is now shared with the channelsWithV2Entitlements test suite and was moved to util.test.ts